project/src/store/api-actions.ts

```diff
@@ -0,0 +1,32 @@
+import { ThunkActionResult } from '../types/action';
+import { loadOffers, requireAuth, changeUser, redirectToRouter } from './actions';
+import { saveToken, Token } from '../services/token';
+import { AuthStatus, ApiRoute, AppRoute } from '../const';
+import { adaptOfferDataToClient } from '../utils';
+import { AuthData ,DataOffer } from '../types/data';
+
+const loadDataOffers = (): ThunkActionResult =>
+  async (dispatch, _getState, api): Promise<void> => {
+    const {data} = await api.get(ApiRoute.Offers);
+    const offers = data.map((item: DataOffer) => adaptOfferDataToClient(item));
+    dispatch(loadOffers(offers));
+  };
+
+const checkAuthAction = (): ThunkActionResult =>
```

Comment:
```suggestion
const checkAuth = (): ThunkActionResult =>
```
у тебя где-то есть в названии action, где-то нету. лучше делай единобразно